scaffold/github/actions/pantheon/review/action.yml

```diff
@@ -107,7 +111,15 @@ runs:
     - name: Deploy to Pantheon
       run: |
         source .github/actions/drainpipe/set-env/bash_aliases
-        drainpipe_exec "terminus workflow:wait ${{ inputs.site-name }}.pr-$DRAINPIPE_PR_NUMBER"
+
+        echo "::notice::Waiting 600 seconds for Pantheon to sync..."
+        drainpipe_exec "terminus workflow:wait -vv --max 600 -- ${{ inputs.site-name }}.pr-$DRAINPIPE_PR_NUMBER"
```

Comment:
We could do this with an environment variable like `TERMINUS_WORKFLOW_WAIT_TIMEOUT=<number>`. In our testing it takes at least 5 minutes to sync even a small site. We'd want to document this variable in the README. Let's have it default to 600 seconds which should cover most cases.